src/js/services/werewolf.js

```diff
@@ -42,4 +42,6 @@ export default new class werewolfService {
     getScript(deck, lang = 'en') { //TODO: better defaults
         return brain.getScriptFromDeck(deck, lang);
     }
+
+    mode = brain.getModes;
```

Comment:
The reason behind the service is to not attach the application to the werewolf-brain. Use the service as a middleware.